data-prepper-plugins/rds-source/src/main/java/org/opensearch/dataprepper/plugins/source/rds/model/ForeignKeyRelation.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.source.rds.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Getter;
+
+@Getter
+@Builder
+public class ForeignKeyRelation {
+    // TODO: add java docs
+    @JsonProperty("database_name")
+    private final String databaseName;
+
+    @JsonProperty("parent_table_name")
+    private final String parentTableName;
+
+    @JsonProperty("referenced_key_name")
+    private final String referencedKeyName;
```

Comment:
can you clarify the difference between this and line 30 ?

Reply:
Foreign key is from the child table. Referenced key is from the parent table, it's the key that foreign key references to.